packaging/rpm/alvr.spec

```
@@ -1,35 +1,52 @@
 Name: alvr
-Version: 16.0.0-rc1
-Release: 1.0.0
+Version: 16.0.0
+Release: 0.0.1rc1
 Summary: Stream VR games from your PC to your headset via Wi-Fi
 License: MIT
-Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v%{version}.tar.gz
+Source: https://github.com/alvr-org/ALVR/archive/refs/tags/v16.0.0-rc1.tar.gz
+URL: https://github.com/alvr-org/ALVR/
 ExclusiveArch: x86_64
-BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel vulkan-headers vulkan-loader-devel
+BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel openssl-devel rpmdevtools rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel selinux-policy-devel vulkan-headers vulkan-loader-devel
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
 Requires: ffmpeg steam
 Requires(post): policycoreutils
 Requires(postun): policycoreutils
+%define alvrBuildDir build/%{name}_server_linux
+# find-debuginfo.sh doesn't appear to be working
 %global debug_package %{nil} 
 
 %description
-ALVR is an open source remote VR display which allows playing SteamVR games on a standalone headset such as Gear VR or Oculus Go/Quest.
-
-%pre
-%define alvrBuildDir build/%{name}_server_linux
+ALVR is an open source remote VR display which allows playing SteamVR games on
+ a standalone headset such as Gear VR or Oculus Go/Quest.
 
 %prep
 %autosetup -D -n %{_builddir}
 
 %build
+# Set CXXFLAGS for ffmpeg
+export CXXFLAGS='-I/usr/include/ffmpeg'
```

Maybe just append instead?
```suggestion
export CXXFLAGS="$CXXFLAGS -I/usr/include/ffmpeg"
```

---

yup my bad, I think I meant to and just forgot, pretty much should always append.